OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -25,6 +25,9 @@
     @NonNull
     private com.google.firebase.analytics.FirebaseAnalytics tracker;
 
+    protected final Logger logger = new Logger(getClass().getName());
```

Comment:
If we go by the book (i.e. our own format of variables ordering) this should be first line after:

````java
public class FirebaseAnalytics implements Analytics {